modules/nf-core/universc/launch/main.nf

```diff
@@ -0,0 +1,62 @@
+process UNIVERSC_LAUNCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the Cell Ranger tool. Please use docker or singularity containers."
+        conda (params.enable_conda ? "hcc::cellranger=3.0.2" : null)
+    }
+    container "tomkellygenetics/universc:1.2.4"
+    containerOptions = "--user root"
```

Comment:
Also not trying to have root in docker containers enforced - any chance we can leave this one out?

Reply:
> Also not trying to have root in docker containers enforced - any chance we can leave this one out?

Unfortunately this is necessary without significant changes to how the container is set up. While the script is running, it modifies files within the dependencies installed. To ensure this is possible, it will check the dependencies are installed in a writeable directory and abort if it cannot pass this check.

This of course does not grant root permissions outside of the container but it is needed to run scripts within it.